frontend/src/homePage.js

```diff
@@ -33,11 +31,13 @@ class Home extends React.Component {
       });
   }
 
+
   // Handles buttons that login & logout user
   handleLoginClick = (e) => {
     const userInfo = fetch("/login")
       .then((response) => response.json())
       .then((info) => {
+
         // If the user has logged in before then the button will now be a logout button
         if (info["showTabStatus"] === true) {
           window.location = this.state.logoutUrl;
```

Comment:
Not in this PR, but below: 

```js
  handleFoldersClick = (e) => {
    window.location = "/myFolders?userKey=" + this.state.userKey;
  };
```

Does this match the design doc?

Reply:
I changed myFolders to /userfolders. Was that the intent of this comment or were asking about how I'm passing userKey as a query parameter?

Reply:
My code now reflects the changes made in the design doc.

Reply:
`/userfolders` is listed as a data-endpoint in the design doc, so it doesn't make sense to redirect the user to it.

My original point was about the capitalization of `MyFolders`.